backend/app/usecases/chat.py

```diff
@@ -161,33 +161,41 @@ def prepare_conversation(
             message_map=initial_message_map,
             last_message_id="",
             bot_id=chat_input.bot_id,
+            should_continue=False,
         )
 
     # Append user chat input to the conversation
     if chat_input.message.message_id:
         message_id = chat_input.message.message_id
     else:
         message_id = str(ULID())
-    new_message = MessageModel(
-        role=chat_input.message.role,
-        content=[
-            ContentModel(
-                content_type=c.content_type,
-                media_type=c.media_type,
-                body=c.body,
-            )
-            for c in chat_input.message.content
-        ],
-        model=chat_input.message.model,
-        children=[],
-        parent=parent_id,
-        create_time=current_time,
-        feedback=None,
-        used_chunks=None,
-        thinking_log=None,
-    )
-    conversation.message_map[message_id] = new_message
-    conversation.message_map[parent_id].children.append(message_id)  # type: ignore
+
+    # TODO: Continueが空メッセージという仕様で良いか要検討
+    # Empty messages for continuity purposes are not added.
```

Comment:
> 空の判定はddbから読み取ったshould_continueから判別可能だと思ったのですが、ここで敢えて空の判定を挟んでいる理由を教えてください（打ち合わせ時に教えていただいたと思うのですが、忘れてしまいました。申し訳ないのですが、ここに記録残していただけると助かります。またそれをコード中のコメントで英語で記載しておいていただけると助かります）
仮に`max_tokens`となって続きがあったとしても、それを無視して別のリクエストを送るケースもあるかもという想定からです。`should_continue`に頼ってしまうとユーザの意図した動作とならない可能性があります。

> 仮に必要な場合は、bedrock claude chatは空メッセージをフロントから受け付けない仕様なので、空の判定で良いと思います！
了解です。

> 空であることの判定は、websocket.pyに記載いただいているものと同様のもので良いと思いましたが、ここでtextかどうかも判定している理由はなんですか？
単に`text`でない場合の仕様をしっかり読み解けておらず、`image`の場合に`body`が空の場合があるのかな？という想定からです。content_tyepが"image"の場合も`body`が空という状況はない感じでしょうか？

> new_messageの直前に、continueボタンによる分岐である旨をコメントで記載お願いします！
分岐の直前にコメントで入れておきます。

Reply:
> 仮にmax_tokensとなって続きがあったとしても、それを無視して別のリクエストを送るケースもあるかもという想定からです。should_continueに頼ってしまうとユーザの意図した動作とならない可能性があります。

ありがとうございます、理解しました！

>単にtextでない場合の仕様をしっかり読み解けておらず、imageの場合にbodyが空の場合があるのかな？という想定からです。content_tyepが"image"の場合もbodyが空という状況はない感じでしょうか？

はい、現在画像添付すると文章入力しないと送れない仕様となっていますので、content_typeによる判定は入れなくて良いかなと思います。

ですが、今後ロジックが変わる可能性もありますので、可読性を上げるため`backend.app.routes.schemas.conversation.py`の`MessageInput`側にロジック記載いただくことは可能ですか？たとえば、

```py
class MessageInput(BaseSchema):
    role: str
    content: list[Content]
    model: type_model_name
    parent_message_id: str | None
    message_id: str | None = Field(
        None, description="Unique message id. If not provided, it will be generated."
    )
    
    def is_content_empty(self) -> bool:
      if len(self.content) == 0:
        raise Exception("No contents"). # This err should not happen
      if self.content[0].body == ''":|
        return True
      return False
```

のようにすると見通しが良いかと思いましたが、いかがでしょうか？